integrationTests/agent.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$EUID" -eq 0 ]
+  then echo "Run without sudo!"
+  exit
+fi
+
+if [ "$1" = "" ]
+  then echo "Missing config file path!"
+  exit
+fi
+
+if [[ $1 != *.json ]]
+  then echo "Config file must be a .json file!"
+  exit
+fi
+
+CONFIG=$PWD/$1
+echo "Config file path: $CONFIG"
+
+if [ "$2" = "" ]
+  then echo "Missing pcap file path!"
+  exit
+fi
+
+if [[ $2 != *.cap ]]
+  then echo "PCAP file must be a .cap file!"
+  exit
+fi
+
+PCAP=$PWD/$2
+echo "PCAP file path: $PCAP"
+
+rm -rf entries/ && mkdir -p entries && rm -rf pprof/* && \
```

Comment:
I assume that you concatenate the commands with `&&` because you want the script to break if one of the commands returns an error.
Instead, add `set -e` at the top of the file, below the hash bang.
It causes bash to exit immediately if a command fails.

Reply:
I'm going to remove this script, this is not being used anymore. I started the development using this script, then switched to starting basenine/api-server/tapper directly from the go test code using exec.command.